hugo.yaml

```diff
@@ -218,25 +218,25 @@ module:
       target: content
       lang: ja
     # fallback pages
-    # - excludeFiles: specs/*
-    #   source: content/en/docs
-    #   target: content/docs
-    #   lang: ja
-    - source: content/en/announcements
-      target: content/announcements
+    - source: content/en
+      target: content
+      excludeFiles: &fallbackExcludeFiles
+        - docs/specs/*
+        - ecosystem/*
+        - site
       lang: ja
 
     ## zh
     - source: content/zh
       target: content
       lang: zh
     # fallback pages
-    - excludeFiles: specs/*
-      source: content/en/docs
-      target: content/docs
+    - source: content/en
+      target: content
+      excludeFiles: *fallbackExcludeFiles
       lang: zh
-    - source: content/en/announcements
-      target: content/announcements
+    - source: content/ja/ecosystem
```

Comment:
```suggestion
    - source: content/en/ecosystem
```

Reply:
Thanks, but no, this _was_ actually meant to be `ja` because the `ja` folder contained a redirect file, but redirects aren't working yet, so I'll implement this later.